roles/core/pxe_stack/readme.rst

```diff
@@ -20,7 +19,8 @@ ip address and filename to use.
 **Files location**
 """"""""""""""""""
 
-* PXE boot files are located in /var/www/html/preboot_execution_environment/.
+* PXE boot files are located in {/var/www/html|/srv/www/htdocs}/preboot_execution_environment/,
```

Comment:
I don't see any way to use a path different than /var/www/html/ in the role yet.
Might be too early to document alternative OS?